### DISCUSSION TITLE
Tkinter MacOS darkmode and titlebar-style

### DISCUSSION THREAD

**Discussion by TomSchimansky** [Saadmairaj/tkmacosx#14] [Q&A] [answered]
Hi,
in the Readme.md of this project are some examples where you have a tkinter-window,
but it's in **MacOS darkmode** like the following:
![Button](https://user-images.githubusercontent.com/66446067/106898252-3ad4a580-66f4-11eb-8045-fa3218a51352.gif)
How can I achieve this, which version of python and tkinter did you use?
Because my windows always stay in light-mode.

Besides this, you also managed to get a tkinter-window without a standard titlebar, which looks very good in my opinion:
![Radiobutton](https://user-images.githubusercontent.com/66446067/106898479-88511280-66f4-11eb-84eb-792847da569b.gif)
How is this possible with tkinter in MacOS?

Thanks for any answers.

**Comment by Saadmairaj** [comment, marked as answer]
For dark mode you can refer to [this answer](https://stackoverflow.com/a/55483871/10364425) on stack overflow. And the second gif is just a window in light mode it is looking like that because the gif quality is not good lol.

**Reply by TomSchimansky** [reply]
Ok, so you used the anaconda python version and it just worked? Or in combination
with `defaults write -g NSRequiresAquaSystemAppearance -bool No` ?

**Reply by Saadmairaj** [reply]
It will work will all versions I guess but there is bug with tkinter's button in dark mode where text color is white by default.

**Reply by wahidawari** [reply]
Hi my name is Wahid Dawari. I am currently learning how to program with python and I am making a guitar trainer. I am using your tuner for an aspect of my project. Can you please email me @ wahid.dawari2003@gmail.com so that I can ask you a couple of questions? I would really appreciate it.

**Comment by wahidawari** [comment]
Hi my name is Wahid Dawari. I am currently learning how to program with python and I am making a guitar trainer. I am using your tuner for an aspect of my project. Can you please email me @ wahid.dawari2003@gmail.com so that I can ask you a couple of questions? I would really appreciate it.
